sdk/core/azure-core/inc/azure/core/response.hpp

```diff
@@ -9,6 +9,9 @@
 #pragma once
 
 #include <azure/core/http/http.hpp>
+#include <azure/core/http/nullable.hpp>
```

Comment:
```suggestion
#include <azure/core/nullable.hpp>
```

Reply:
Ouch - just a wee bit of copy-pasta here.